pints/_mcmc/_emcee_hammer.py

```diff
@@ -160,7 +166,7 @@ def tell(self, fx):
         accepted = np.array([False] * self._n_chains)
         r_log = np.log(np.random.rand())
         q = (
-            (self._n_chains - 1) * np.log(self._z)
+            (self._n_parameters - 1) * np.log(self._z)
             + proposed_log_pdf[0] - self._current_log_pdfs[self._k])
```

Comment:
Could you explain this q to me? Is this the log q of the q mentioned in the docstring
```
q = z^{N - 1} p(Y) / p(X_k(t))?
```
If so, maybe `log_q` would be more clear? What do you think?

In the docstring it also says: ``N`` is the number of chains (or walkers). So do you really want to use the number of parameters, or is the docstring wrong?

Reply:
Renamed to `log_q`: good idea! And, yes, good spot in the docstrings: they were wrong and have now been corrected.